manifests/akv-manifests.yaml

```diff
@@ -0,0 +1,76 @@
+- kind: Node
```

Comment:
Since you named this file `akv-manifests.yaml` I would expect only akv specific examples to be added here. So that either you rename the file or create different files to encapsulate different type of QKM components such as Roles, Nodes,....

Reply:
These files are no longer use, replaced with new format ones